README.md

```diff
@@ -4,6 +4,10 @@
 
 Deployments are triggered automatically when pushing to the *master* branch.
 
+## [Code of Conduct](https://github.com/mtlpy/code-of-conduct)
+
+Montréal-Python has adopted a Code of Conduct that we expect project participants to adhere to. If you need more informations please read [the full text](https://github.com/mtlpy/code-of-conduct) for an overview of the types of behaviours deemed inappropriate. Please read so that you can understand what actions will and will not be tolerated.
```

Comment:
information* (it’s uncountable).

Maybe break lines at a reasonable column (72 or 99)?